source/_lovelace/entities.markdown

```diff
@@ -11,28 +11,28 @@ To add the Entities card to your user interface, click the Lovelace menu (three
 {% configuration %}
 type:
   required: true
-  description: entities
+  description: "`entities`"
   type: string
 entities:
   required: true
   description: "A list of entity IDs or `entity` objects or special row objects (see below)."
   type: list
 title:
   required: false
-  description: The card title.
+  description: Card title
   type: string
 icon:
   required: false
   description: An icon to display to the left of the title
   type: string
 show_header_toggle:
   required: false
-  description: Button to turn on/off all entities.
+  description: Button to turn on/off all entities
```

Comment:
ideally, every line with more than 3 words, so end with a full stop.

Both documentation guides by Microsoft's & Google agree on that. So I would rather add it every, where, than remove it.